parsers/MY_WM.py

```diff
@@ -23,18 +23,22 @@
 DOMAIN = 'www.gso.org.my'
 PRODUCTION_THRESHOLD = 10 # MW
 TIMEZONE = 'Asia/Kuala_Lumpur'
-URL_FORMAT_STRING = 'https://{}/SystemData/{}/GetChartDataSource'
-CONSUMPTION_URL = URL_FORMAT_STRING.format(DOMAIN, 'SystemDemand.aspx')
-EXCHANGE_URL = URL_FORMAT_STRING.format(DOMAIN, 'TieLine.aspx')
-PRODUCTION_URL = URL_FORMAT_STRING.format(DOMAIN, 'CurrentGen.aspx')
+CONSUMPTION_URL \
+    = f'https://{DOMAIN}/SystemData/SystemDemand.aspx/GetChartDataSource'
```

Comment:
Why the `\`, isn't the line short enough to not do this?
```suggestion
CONSUMPTION_URL = f'https://{DOMAIN}/SystemData/SystemDemand.aspx/GetChartDataSource'
```